### ISSUE TITLE
Achievements are not durable

### ISSUE THREAD

**Issue by efeberku** [YoYoGames/GMEXT-GooglePlayServices#7]
I am using:
GooglePlayServices_Achievements_Unlock("achievement_id");

But after a some time or a day later achievement becomes locked again. How can we save the achievement state?

https://github.com/playgameservices/android-basic-samples/blob/f6a39dfcac2f37c2586626057fb79bec7b610211/TypeANumber/src/main/java/com/google/example/games/tanc/MainActivity.java#L510

Other extensions uses googleSignInAccount why our extension do not have this?

https://developers.google.com/android/reference/com/google/android/gms/games/Games#getAchievementsClient(android.app.Activity,%20com.google.android.gms.auth.api.signin.GoogleSignInAccount)

**Comment by efeberku** [comment]
I imported these and have no issue now. (I don't know if it fixed the GooglePlayServices_Achievements_GetStatus(0); no network error.)

import com.google.android.gms.games.AchievementsClient;
import com.google.android.gms.games.achievement.Achievement;
import com.google.android.gms.games.achievement.AchievementBuffer;
import com.google.android.gms.games.achievement.AchievementEntity;

**Comment by DiasFranciscoA** [comment]
If you marked yourself as developer is normal that they are not durable.. developers will have their acheivements reset after a couple days so you can properly test various use cases. Is this what's happening or is it happening on production?

**Comment by efeberku** [comment]
Thank you for responding. I solved the issue. (It was a test account issue I think.) I added myself as a test user and the problem solved.

I also imported the things above in the java side.

I also deleted the GooglePlayServices_Achievements_GetStatus(); code from my game.

My only problem is GooglePlayServices_Achievements_GetStatus(); crashes some phones. But I didn't tried after I imported AchievementsClient etc. (Maybe that was the reason but I will not try it.)
